.github/workflows/php.yml

```
@@ -129,7 +133,7 @@ jobs:
 
     - name: Run PHPUnit
       working-directory: src/${{ matrix.project }}
-      run: vendor/bin/phpunit --testsuite integration --testsuite unit --coverage-text --coverage-clover=coverage.clover
+      run: vendor/bin/phpunit --coverage-text --coverage-clover=coverage.clover
```

I think the phpt failures are due to xdebug, can you try it with:
```suggestion
      run: XDEBUG_MODE=coverage vendor/bin/phpunit --coverage-text --coverage-clover=coverage.clover
```